test/empty-options.test.js

```diff
@@ -16,14 +16,14 @@ describe('when applied with no options', () => {
     const compilerEnv = pluginEnvironment.getEnvironmentStub();
     compilerEnv.context = '';
 
-    const plugin = new UglifyJsPlugin();
+    const plugin = new UglifyJsPlugin({ parallel: { cache: false, workers: 0 } });
     plugin.apply(compilerEnv);
     eventBindings = pluginEnvironment.getEventBindings();
   });
 
   it('matches snapshot', () => {
     const compiler = createCompiler();
-    new UglifyJsPlugin().apply(compiler);
+    new UglifyJsPlugin({ parallel: { cache: false } }).apply(compiler);
```

Comment:
Please remove this change here and just make a new test inside `parallel-option.test.ja`. Something like:
  it('matches snapshot', () => {
    const compiler = createCompiler();
    new UglifyJsPlugin({ parallel: { cache: false } }).apply(compiler);

    return compile(compiler).then((stats) => {
      const errors = stats.compilation.errors.map(cleanErrorStack);
      const warnings = stats.compilation.warnings.map(cleanErrorStack);

      expect(errors).toMatchSnapshot('errors');
      expect(warnings).toMatchSnapshot('warnings');

      for (const file in stats.compilation.assets) {
        if (Object.prototype.hasOwnProperty.call(stats.compilation.assets, file)) {
          expect(stats.compilation.assets[file].source()).toMatchSnapshot(file);
        }
      }
    });
  });